### PR TITLE
Added the version 2.9.2 changelog

### PR DESCRIPTION
**Pull request by sb230132**
Update the release notes with the 2.9.2 changelog.

# Description

[Short summary of why you created this PR]

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: timescaledb/overview/release-notes/index.md [hunk 1]
```diff
@@ -21,6 +21,20 @@ Want to stay up-to-date with new releases? Subscribe to get
 * Introduce fixed schedules for background jobs and the ability to check job errors.
 * Use `alter_data_node()` to change the data node configuration. This function introduces the option to configure the availability of the data node.
 
+## 2.9.2 (2023-01-26)
```
Comment (iroussos) [inline timescaledb/overview/release-notes/index.md:24]: Why does the [`CHANGELOG.md` in the 2.9.x branch](https://github.com/timescale/timescaledb/pull/5210/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR12) has a different date than the release notes in this PR and the [canonical Changelog in the main project](https://github.com/timescale/timescaledb/blob/main/CHANGELOG.md#292-2023-01-26)?

Do we need to revisit the branching process for release branches to make sure that no changes are introduced or are you going to reverse rebase/merge afterwards?

**Review by iroussos — COMMENTED** [review]
Looks good to me @bharathy,

Other my other comment on the date: 
- why haven't we included https://github.com/timescale/timescaledb/pull/5172 in this list and the changelog? I can see that we cherrypicked the full fix by Fabrizio, or am I missing something here?
- I can see more PRs [labeled as 2.9.2](https://github.com/timescale/timescaledb/milestone/66?closed=1), did we forget to add anything?

File: timescaledb/overview/release-notes/index.md (hunk 1 above)
Comment (sb230132) [reply timescaledb/overview/release-notes/index.md]: @iroussos 
Changes of #5172 is available in release PR (#5120), however its not mentioned in changelog. According to @svenklemm we don't specify PR's which fix only testfiles. Thus you see few missing PRs when compared against
https://github.com/timescale/timescaledb/milestone/66

Fixed the date. Thanks.

Comment (iroussos) [reply timescaledb/overview/release-notes/index.md]: Thank you @sb230132, you are right I forgot about that! Everything looks good to me, approving on my side :-)

**Review by iroussos — APPROVED** [review]

**Pull request merged**